api/v1alpha1/clustergroupupgrade_types.go

```diff
@@ -33,6 +33,12 @@ type RemediationStrategySpec struct {
 	Timeout int `json:"timeout,omitempty"`
 }
 
+// BlockingCR defines the Upgrade CRs that block the current CR from running if they are currently running
```

Comment:
nit. maybe reword. 
BlockingCR defines the running Upgrade CRs that block the current CR from running